src/gdpr-utils.js

```diff
@@ -79,13 +79,21 @@ export function hasOptedIn(token, options) {
  * @param {string} [options.persistenceType] Persistence mechanism used - cookie or localStorage
  * @param {string} [options.persistencePrefix=__mp_opt_in_out] - custom prefix to be used in the cookie/localstorage name
  * @param {boolean} [options.ignoreDnt] - flag to ignore browser DNT settings and always return false
+ * @param {boolean} [options.debug] - flag to show debug statements
  * @returns {boolean} whether the user has opted out of the given opt type
  */
 export function hasOptedOut(token, options) {
     if (_hasDoNotTrackFlagOn(options)) {
+        if (options.debug) {
```

Comment:
I believe the `console` shim in utils.js already takes care of this logic for you, so you wouldn't have to thread `debug` through the params here. (it also protects against the absence of `window.console` in browsers that no longer matter, ha)